src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java

```diff
@@ -0,0 +1,191 @@
+
+import io.forty11.j.J;
+import io.forty11.sql.Sql;
+import io.forty11.web.Url;
+import io.rcktapp.api.*;
+
+import io.rcktapp.api.Collection;
+import io.rcktapp.api.service.Service;
+
+import io.rcktapp.rql.Replacer;
+import io.rcktapp.rql.Rql;
+import io.rcktapp.rql.Stmt;
+import io.rcktapp.rql.sql.SqlRql;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class SqlGetHandlerTest {
+
+    @Mock
+    Service service;
+
+    @Mock
+    Api api;
+
+    @Mock
+    Endpoint endpoint;
+
+    @Mock
+    Action action;
+
+    @Mock
+    Chain chain;
+
+    @Mock
+    Response response;
+
+    @Mock
+    SqlDb sqlDb;
+
+    @Mock
+    SqlRql sqlRql;
+
+    @Mock
+    Stmt stmt;
+
+    @Mock
+    Connection connection;
+
+    @Mock
+    SqlGetHandler handler;
+
+    @Captor
+    ArgumentCaptor<List> queryParametersCaptor;
+
+    @Captor
+    ArgumentCaptor<String> sqlCaptor;
+    @Captor
+    ArgumentCaptor<Map<String, String>> rqlParamsCaptor;
+
+    @Test
+    public void testSqlGetHandler_queryParametersAreProperlyPassedToQueryObjectsMethod() throws Exception {
+
+        Url url = new Url("http://localhost:8080/api/lift/us/plugins/updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)");
+
+        String passedQuery = "updatexml(rand(),concat(CHAR(126),user(),CHAR(126)),null)";
+        String collectionName = "plugins";
+        String expectedQuery = " SELECT * FROM " + collectionName + " WHERE plugins IN (?,?,?,?,?) ";
+        String[] inClause = {"updatexml(rand()", "concat(CHAR(126)", "user()", "CHAR(126))", "null)"};
+
+        // Setup request object and sql table objects to properly reflect
+        Request request = new Request(url, "GET", new HashMap<>(), new HashMap<>(), "");
+        Service.ApiMatch apiMatch = new Service.ApiMatch(api, endpoint, "GET", request.getUrl(), "http://localhost:8080/api/lift/us/", collectionName + "/" + passedQuery + "/");
+        Collection collection = new Collection();
+        Entity entity = new Entity(new Table(sqlDb, collectionName));
+        Attribute attribute = new Attribute();
+
+        attribute.setName(collectionName);
+        entity.setKey(attribute);
+        collection.setEntity(entity);
+        request.setApiMatch(apiMatch);
+
+        // Mocked methods that return local objects used in the future
+        when(chain.getService()).thenReturn(service);
+        when(service.getDb(request.getApi(), request.getCollectionKey(), SqlDb.class)).thenReturn(sqlDb);
+        when(sqlDb.getConnection()).thenReturn(connection);
+
+        try (MockedStatic<Rql> rqlMockedStatic = mockStatic(Rql.class)) {
+            try (MockedStatic<Sql> sqlMockedStatic = mockStatic(Sql.class)) {
```

Comment:
nitpick: you could have both of these in the same try block with 
```
try (MockedStatic<Rql> rqlMockedStatic = mockStatic(Rql.class); 
MockedStatic<Sql> sqlMockedStatic = mockStatic(Sql.class)) {}
```

Reply:
nesting try like this is a smell, please do change it